neurodsp/tests/test_aperiodic_dfa.py

```diff
@@ -0,0 +1,20 @@
+"""Tests for fractal analysis using fluctuation measures."""
+
+from neurodsp.tests.settings import FS
+
+from neurodsp.aperiodic.dfa import *
+
+###################################################################################################
+###################################################################################################
+
+def test_compute_fluctuations(tsig):
+
+    t_scales, flucs, exp = compute_fluctuations(tsig, FS)
```

Comment:
what's tsig?

Reply:
It's just a pre-computed test signal. It's just a small optimization, from pytest, to not have to generate data objects every time you need one. 

It's generated in `conftest.py`, from:
```
@pytest.fixture(scope='session')
def tsig():

    yield np.random.randn(1000)
```